web/src/AutoBoundariesMode.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+  import BackButton from "./BackButton.svelte";
+  import {
+    FillLayer,
+    GeoJSON,
+    LineLayer,
+    hoverStateFilter,
+    type LayerClickInfo,
+  } from "svelte-maplibre";
+  import { Link, layerId } from "./common";
+  import { isLine } from "svelte-utils/map";
+  import { SplitComponent } from "svelte-utils/top_bar_layout";
+  import { app, mode, autosave } from "./stores";
+
+  let gj = JSON.parse($app!.renderAutoBoundaries());
+
+  function add(e: CustomEvent<LayerClickInfo>) {
+    let name = window.prompt("What do you want to name the neighbourhood?");
+    if (!name) {
+      return;
+    }
+    try {
+      // TODO Waypoints will be missing; editing will not work
```

Comment:
Will be fixed by switching to the new area drawing tool, which is smart about handling freehand waypoints